docs/admin-guide/configuration/uploads.md

```diff
@@ -1,27 +1,23 @@
+Defines the directory to use for (temporarily) storing uploaded files such as product images, shipment documents, and data imports.
+If the uploads folder does not exist, tomcat will attempt to create it (which will only succeed if tomcat has write permissions in the parent location).
 
-## File Uploads Location
-Directory for storing temporary uploaded files, including but not limited to product images, 
-shipment documents, data imports.
+## Configuration
 
 ```shell
 openboxes:
     uploads:
         location: "uploads"
 ```
 
-### Customization
+## Customization
+The default configuration adds the uploads directory to the root of the server. This is fine for local testing, but in production you'll likely want to override this behaviour to provide a more specific path for the uploads directory.
 
-```shell
-openboxes:
-    uploads:
-        location: "uploads"
-```
+Something like `"/opt/tomcat/uploads"` would likely work, or `"/opt/tomcat/webapps/openboxes/uploads"` if you have multiple tomcat apps.
```

Comment:
We also need to create the directory (which is the reason why "uploads" doesn't work). See some of the suggestions from the community forum posts in case there's something there that you can copy and paste. 

https://community.openboxes.com/t/unable-to-upload-file-due-to-exception-uploads-inventory-xls-no-such-file-or-directory/211
https://community.openboxes.com/t/filenotfoundexception-tp-link-eap245-ac1750-wireless-gigabit-ceil-eap245-jpg-permission-denied/281